src/Components/Explore/Explore.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import style from './Explore.module.scss';
+import image from './exploreimg.png';
+
+export default function Explore() {
```

Comment:
I have written an eslint rule to throw warning for the rule `react/function-component-definition`

Rule definition states that

> This rule is aimed to enforce consistent function types for function components. By default it prefers function declarations for named components and function expressions for unnamed components.

And by convention the function definition is preferred to be an arrow function.

So, first create an arrow function

```jsx
const Explore = () => {}
```

and then export it in the end

```jsx
export default Explore;
```